docs/actions-php.md

```diff
@@ -0,0 +1,120 @@
+#
+-->
+
+## Creating and invoking PHP actions
+
+The process of creating PHP actions is similar to that of [other actions](actions.md#the-basics).
+The following sections guide you through creating and invoking a single PHP action,
+and demonstrate how to bundle multiple PHP files and third party dependencies.
+
+PHP actions are executed using PHP 7.1.18.
+To use this runtime, specify the `wsk` CLI parameter `--kind php:7.1` when creating or updating an action.
+This is the default when creating an action with file that has a `.php` extension.
+
+An action is simply a top-level PHP function. For example, create a file called `hello.php`
+with the following source code:
+
+```php
+<?php
+function main(array $args) : array
+{
+    $name = $args["name"] ?? "stranger";
+    $greeting = "Hello $name!";
+    echo $greeting;
+    return ["greeting" => $greeting];
+}
+```
+
+PHP actions always consume an associative array and return an associative array.
+The entry method for the action is `main` by default but may be specified explicitly when creating
+the action with the `wsk` CLI using `--main`, as with any other action type.
+
+You can create an OpenWhisk action called `helloPHP` from this function as follows:
+
+```
+wsk action create helloPHP hello.php
+```
+
+The CLI automatically infers the type of the action from the source file extension.
+For `.php` source files, the action runs using a PHP 7.1 runtime.
+
+Action invocation is the same for PHP actions as it is for [any other action](actions.md#the-basics).
+
+```
+wsk action invoke --result helloPHP --param name World
+```
+
+```json
+{
+  "greeting": "Hello World!"
+}
+```
+
+Find out more about parameters in the [Working with parameters](./parameters.md) section.
+
+## Packaging PHP actions in zip files
+
+You can package a PHP action along with other files and dependent packages in a zip file.
+The filename of the source file containing the entry point (e.g., `main`) must be `index.php`.
+For example, to create an action that includes a second file called `helper.php`,
+first create an archive containing your source files:
+
+```bash
+zip -r helloPHP.zip index.php helper.php
+```
+
+and then create the action:
+
+```bash
+wsk action create helloPHP --kind php:7.1 helloPHP.zip
+```
+
+## Including Composer dependencies
+
+If your PHP action requires [Composer](https://getcomposer.org) dependencies,
+you can install them as usual using `composer require` which will create a `vendor` directory.
+Add this directory to your action's zip file and create the action:
+
+```bash
+zip -r helloPHP.zip index.php vendor
+wsk action create helloPHP --kind php:7.1 helloPHP.zip
+```
+
+The PHP runtime will automatically include Composer's autoloader for you, so you can immediately
+use the dependencies in your action code.
+
+Note that if you don't include your own `vendor` folder, then the runtime will include one for you.
+The following PHP extensions are available in addition to the standard ones:
```

Comment:
Need to separate PHP extensions list from the Composer info as they aren't the same thing. Composer manages PHP components, PHP extensions are compiled C libraries that are loaded by the language and the the user cannot add to or remove them.

Reply:
thanks @akrabat for the review - I think addressed in my last commit if you can kindly look again.

Reply:
Yes. That's great. Thanks.